RecoLocalCalo/HcalRecAlgos/src/parseHBHEPhase1AlgoDescription.cc

```diff
@@ -9,6 +9,35 @@
 // Phase 1 HBHE reco algorithm headers
 #include "RecoLocalCalo/HcalRecAlgos/interface/SimpleHBHEPhase1Algo.h"
 
+static std::unique_ptr<DoMahiAlgo>
+parseHBHEMahiDescription(const edm::ParameterSet& conf)
```

Comment:
To follow the naming choice of previous methods (M0, M1, M2, M3), at this layer the text should say M4.